GLMakie/src/glshaders/particles.jl

```diff
@@ -75,6 +75,24 @@ function draw_mesh_particle(screen, p, data)
         texturecoordinates = nothing
     end
 
+    # TODO: use instance attributes
+    if to_value(data[:uv_transform]) isa Vector
+        transforms = pop!(data, :uv_transform)
+        @gen_defaults! data begin
+            uv_transform = map(transforms) do transforms
+                # 3x Vec2 should match the element order of glsl mat3x2
+                output = Vector{Vec2f}(undef, 3 * length(transforms))
+                for i in eachindex(transforms)
+                    output[3 * (i-1) + 1] = transforms[i][Vec(1, 2)]
+                    output[3 * (i-1) + 2] = transforms[i][Vec(3, 4)]
+                    output[3 * (i-1) + 3] = transforms[i][Vec(5, 6)]
+                end
+                return output
+            end => TextureBuffer
+        end
+    else 
+        # handled automatically
+    end
```

Comment:
This is a bit more complicated because we use textures rather than instance buffers. Textures don't support matrix inputs and only up to 4 component vectors, so I'm repacking the matrix into Vec2f's here. Those should match up with memory order of mat3x2 in glsl if I got things correct.